Project.toml

```diff
@@ -4,6 +4,7 @@ version = "0.3.0"
 
 [deps]
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
```

Comment:
so as not to clutter the git log, could you please rebase 161a6a84 with bb2557e?